text/0000-crates.io-default-ranking.md

```diff
@@ -0,0 +1,1443 @@
+        <td>693</td>
+        <td>11301</td>
+        <td>17%, 🌧</td>
+        <td>80%, 🌤</td>
+    </tr>
+    <tr>
+        <td>combine</td>
+        <td>10,809</td>
+        <td>4,252</td>
+        <td>1,115</td>
+        <td>3026</td>
+        <td>5%, 🌧</td>
+        <td>60%, 🌥</td>
+    </tr>
+    <tr>
+        <td>lalrpop</td>
+        <td>7,108</td>
+        <td>1,928</td>
+        <td>796</td>
+        <td>1767</td>
+        <td>3%, 🌧</td>
+        <td>40%, 🌧</td>
+    </tr>
+    <tr>
+        <td>peresil</td>
+        <td>8,960</td>
+        <td>1,859</td>
+        <td>427</td>
+        <td>1710</td>
+        <td>3%, 🌧</td>
+        <td>20%, 🌧</td>
+    </tr>
+</table>
+
+### Credibility
+
+We think credibility is an even more subjective measure than quality. We
+considered using number of other crates an author has, but that would skew
+heavily towards [retep998][]. Highlighting Rust team members is also a
+possibility since people tend to regard them more highly, but there are many
+crate authors who are not on any Rust team who are releasing excellent crates.
+We have [an idea for a more personal "favorite authors" list][favs] that we
+think would help indicate credibility. With this proposed feature, each person
+can define credibility for themselves, which makes this measure less gameable
+and less of a popularity contest.
+
+[retep998]: https://crates.io/users/retep998
+[favs]: https://github.com/rust-lang/crates.io/issues/494
+
+### Overall
+
+Since documentation/ease of use was such a highly mentioned factor in peoples' decisions, we propose that, instead of averagaing the three scores, we weight ease of use by 2x and divide by 4 instead of 3.
+
+<table>
+    <tr>
+        <th>Crate</th>
+        <th>Ease of use</th>
+        <th>Maintenance</th>
+        <th>Popularity</th>
+        <th>Overall</th>
+    </tr>
+    <tr>
+        <td>nom</td>
+        <td>92%, ☀️</td>
+        <td>78%, ⛅️</td>
+        <td>100%, ☀️</td>
+        <td>91%, ☀️</td>
+    </tr>
+    <tr>
+        <td>combine</td>
+        <td>103%, ☀️</td>
+        <td>68%, 🌥</td>
+        <td>60%, 🌥</td>
+        <td>84%, 🌤</td>
+    </tr>
+    <tr>
+        <td>peg</td>
+        <td>66%, 🌥</td>
+        <td>100%, ☀️</td>
+        <td>80%, 🌤</td>
+        <td>78%, ⛅️</td>
+    </tr>
+    <tr>
+        <td>lalrpop</td>
+        <td>80%, 🌤</td>
+        <td>53%, ☁️</td>
+        <td>40%, 🌧</td>
+        <td>63%, 🌥</td>
+    </tr>
+    <tr>
+        <td>peresil</td>
+        <td>105%, ☀️</td>
+        <td>3%, 🌧</td>
+        <td>20%, 🌧</td>
+        <td>58%, ☁️</td>
+    </tr>
+</table>
+
+
+## Display
+
+On a list of crates, the letter representing the score in each category plus
+the overall score would be displayed using text, color, symbols, and detail on
+hover, with a link to a more thorough explanation. We like the information
+density in the way [npms][] displays scores:
+
+<img src="http://i.imgur.com/yadRNyy.png" src="Example npms score circles" width="442" />
```

Comment:
Markdown syntax please. At least you’ll avoid nonsensical HTML where alt text goes into the `src` attribute :)

```
![Example npms score circles](http://i.imgur.com/yadRNyy.png)
```